includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -434,43 +471,92 @@ protected function setup_assets() {
 			new Script(
 				'googlesitekit-modules-reader-revenue-manager',
 				array(
-					'src'          => $base_url . 'js/googlesitekit-modules-reader-revenue-manager.js',
-					'dependencies' => array(
+					'src'           => $base_url . 'js/googlesitekit-modules-reader-revenue-manager.js',
+					'dependencies'  => array(
 						'googlesitekit-vendor',
 						'googlesitekit-api',
 						'googlesitekit-data',
 						'googlesitekit-modules',
+						'googlesitekit-notifications',
 						'googlesitekit-datastore-site',
 						'googlesitekit-datastore-user',
 						'googlesitekit-components',
 					),
+					'load_contexts' => array( Asset::CONTEXT_ADMIN_SITEKIT, Asset::CONTEXT_ADMIN_POST_EDITOR ),
 				)
 			),
 		);
 
 		if ( Feature_Flags::enabled( 'rrmModuleV2' ) ) {
 			$assets[] = new Script(
-				'googlesitekit-reader-revenue-manager-block-editor-plugin',
+				'blocks-reader-revenue-manager-block-editor-plugin',
 				array(
 					'src'           => $base_url . 'js/blocks/reader-revenue-manager/block-editor-plugin/index.js',
 					'dependencies'  => array(
 						'googlesitekit-data',
 						'googlesitekit-i18n',
 						'googlesitekit-modules',
+						'googlesitekit-modules-reader-revenue-manager',
```

Comment:
I found that the main RRM asset was already being loaded on the post edit page when the Admin Bar was enabled, and it seemed to cause a problem when it wasn't there - so I've made it a dependency here, and was able to remove the call to `registerStore( Data )` from each block entry point.